app/routers/upbit_trading.py

```
@@ -0,0 +1,496 @@
+# ===== Helper Functions =====
+
+async def cancel_existing_buy_orders(market: str):
+    """해당 마켓의 기존 매수 주문들을 취소"""
+    try:
+        open_orders = await upbit.fetch_open_orders(market)
+        buy_orders = [order for order in open_orders if order.get('side') == 'bid']
+
+        if buy_orders:
+            order_uuids = [order['uuid'] for order in buy_orders]
+            await upbit.cancel_orders(order_uuids)
+    except:
```

Ruff (0.14.2)

Do not use bare `except`

(E722)